app/assets/js/app/main.js

```diff
@@ -4,3 +4,4 @@ import './modules/accordion'
 import './modules/analytics'
 import './modules/inpagelink'
 import './modules/focus-styles'
+// import './modules/household'
```

Comment:
@jonnyshaw89 This needs to remain commented out for now. It should be uncommented when we work on **Household Composition Entry: Realtime Changes** story.